package.json

```diff
@@ -3,13 +3,17 @@
   "license": "MIT",
   "version": "1.0.0",
   "description": "A tool to update a beacon server value on a time interval",
+  "engines": {
+    "node": "^14.17.1"
+  },
   "main": "./dist/src/start.js",
   "files": [
     "dist",
     "serverless.yml",
     "*.js"
   ],
   "scripts": {
+    "postinstall": "git clone https://github.com/api3dao/airnode.git airnode; cd airnode && yarn run bootstrap && yarn build && cd ..; cp -a airnode/node_modules/@api3/airnode-node/. node_modules/@api3/airnode-node; rm -rf airnode",
```

Comment:
I'm not sure about this tbh. If there's no simple way to do this I wouldn't force it.
Since we all have the airnode cloned anyway what about this. We'll remove the postinstall so nothing is done automatically but we add `link:airnode` target that would expect a path (both relative and absolute) to the local `airnode` directory and links the `airnode/node_modules/@api3` to `node_modules/@api3`? It would require a tiny node script to do that but that's like 5 lines of code. WDYT?

Reply:
I can make these changes, but this wouldn't work for github tests, right? :thinking:

Reply:
:thinking: You're right, that's true. Ok, let's go with your approach, it's temporary anyway.

Reply:
or alternatively request a 0.5.1 release for airnode-node? :smile:

Reply:
Unfortunately, I think this is not the last time we're having this problem while we don't have something at least semi-stable. We'll see.

Reply:
That's a great point and makes sense to add